okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/RemotingOkHttpCall.java

```diff
@@ -101,6 +101,9 @@ public void onResponse(Call call, Response response) throws IOException {
         });
 
         try {
+            // We don't enforce a timeout here because it's not possible to know how long this operation might take.
+            // First, it might get queued indefinitely in the Dispatcher, and then it might get retried a (potentially)
+            // unknown amount of times by the BackoffStrategy.
```

Comment:
plus: This call times out when the underlying OkHttp call times out, possibly after a number of retries.

right?

Reply:
Would you rather I said only that? There is some overlap with my comment. Feel free to modify it as you wish & merge.